app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -1,4 +1,34 @@
 class ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate < OrchestrationTemplate
+  def parameter_groups
+    [OrchestrationTemplate::OrchestrationParameterGroup.new(
+      :label      => "vApp Parameters",
+      :parameters => vapp_parameters,
+    )]
+  end
+
+  def vapp_parameters
+    [
+      OrchestrationTemplate::OrchestrationParameter.new(
+        :name          => "deploy",
+        :label         => "Deploy vApp",
+        :data_type     => "boolean",
+        :default_value => false,
```

Comment:
@gberginc where would an orchestration template be used if not deploying a vApp?  Wondering what this option is used for.